tests/production_test.go

```diff
@@ -12,12 +13,14 @@ import (
 )
 
 func TestProductionLogger(t *testing.T) {
+	ctx := context.Background()
 	var output bytes.Buffer
 
-	log, err := logger.ConfigureProductionLogger("info", &output)
+	ctx, err := logger.ConfigureProductionLogger(ctx, "info", &output)
```

Comment:
i feel like this function should still return the `Logger` so that users of this library know to handle it and call log.Sync().

Having to get it from the context is not obvious.

something like
```suggestion
	ctx, log, err := logger.ConfigureProductionLogger(ctx, "info", &output)
```

Reply:
Yep I agree with what you are saying that it is hard to get the logger - that's why I am writing some example documentation right now so that people know to get it from the context. Calling `logger.Sync()` may not always take place in the same context where `ConfigureProductionLogger`, so I don't think it's appropriate to return the logger from the method.

Reply:
@tim-thacker-nullify can you check the latest commit? I have added in documentation to the `README`

Reply:
looks good